app/lib/import_export_constants.rb

```diff
@@ -495,4 +495,32 @@ def self.rct_node(schema, label, fields)
     transfers: TRANSFERS_EXPORT_OPTIONS,
     histories: HISTORIES_EXPORT_OPTIONS
   }.freeze
+
+  FULL_HISTORY_PATIENT_CONFIG = {
+    format: 'xlsx',
+    separate_files: false,
+    data: {
+      patients: {
+        checked: FULL_HISTORY_PATIENTS_FIELDS,
+        headers: FULL_HISTORY_PATIENTS_HEADERS,
+        tab: 'Monitorees List'
+      },
+      # assessment fields and headers need to be duplicated because they may be modified
+      assessments: {
+        checked: FULL_HISTORY_ASSESSMENTS_FIELDS.dup,
```

Comment:
Are you saying that it's getting double duped?

Reply:
The reason this was being duped originally is because these fields in the config get modified during the export job depending on which symptoms are present. By moving this to a constant in a scope outside of the `full_history_patient` method, wouldn't any modifications to the config continue to persist after the end of `full_history_patient`?